apiary.apib

```diff
@@ -1586,6 +1586,7 @@ Get a list of deals.
                         + contact
                 + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
             + phase_id: `1758e847-cc1f-04ce-bd17-5a852e8c01b4` (string, optional)
+            + estimated_closing_date: `2018-02-11T16:45:30+00:00` (string, optional, nullable)
```

Comment:
In the body, it doesn't contain the time, so I would also not filter with it.

Reply:
Alright